src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -1960,6 +1960,38 @@ public static IEnumerable<object[]> XmlDictionaryReaderQuotasData
         }
     }
 
+    [Fact]
+    public static void DCS_XmlElementAsRoot()
+    {
```

Comment:
These tests don't pass in Net Native yet. I'll followup to fix them in Net Native.